src/lib.rs

```diff
@@ -125,6 +132,33 @@ fn find_all_links(config: &Config) -> Vec<MarkupLink> {
     links
 }
 
+fn find_git_ignored_files() -> Option<Vec<PathBuf>> {
+    let output = Command::new("git")
```

Comment:
What if 'git' is not installed on the system? I guess some kind of early failure with good error message is a must. Also I would love to add this info to the docs/readme that 'git' is a required binary dependency for this to work